lib/acl.js

```diff
@@ -12,11 +12,20 @@ var internals = {};
  *
  * @param user	- The user to check if they have a role
  * @param role	- The role to check if the user has
+ * @param hierarchy	- The configured hierarchy
+ * @param request - The Hapi request object
+ * @param reply - The Hapi reply object
  * @returns {*}
  */
-exports.checkRoles = function(user, role, hierarchy) {
+exports.checkRoles = function(user, role, hierarchy, request, reply, errCallback) {
```

Comment:
It was to hand back the request lifecycle to the implementor when 'errCallback' was called...

But since moved it to onPreHandler where request/reply are available...and seemingly more appropriate to call the forbiddenHandler as well